src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java

```diff
@@ -97,7 +97,7 @@ private void addEntryDataFromPDDocumentInformation(File pdfFile, BibEntry entry)
                         // default time stamp follows ISO-8601. Reason: https://xkcd.com/1179/
                         String date = new SimpleDateFormat("yyyy-MM-dd")
                                 .format(creationDate.getTime());
-                        appendToField(entry, InternalBibtexFields.TIMESTAMP, date);
+                        appendToField(entry, Globals.prefs.get(JabRefPreferences.TIME_STAMP_FIELD), date);
```

Comment:
I agree. Moving ahead to date-added, date-modified (#1575, comment and https://github.com/koppor/jabref/issues/130) will sort this out.